great_tables/_gt_data.py

```diff
@@ -396,6 +396,47 @@ def reorder(self, vars: list[str]) -> Self:
 
         return self[new_order]
 
+    # Get a mutated version of the Boxhead that:
+    # - removes hidden columns
+    # - places the stub column in the correct position
+    # - places the row group column in the correct position (if option is set for that)
+    def final_columns(self, options: Options) -> Self:
+
+        # Remove hidden columns from the boxhead
+        visible_columns = [x for x in self._d if x.visible]
+
+        # If there is a row group column, place it at the start of the boxhead
+        stub_column = [x for x in self._d if x.type == ColInfoTypeEnum.stub]
+
+        if len(stub_column) != 0:
+            # Get index of stub column and use that to move it to the start of the boxhead list
+            stub_index = self._d.index(stub_column[0])
+            visible_columns = [self._d[stub_index]] + [
```

Comment:
Don't you already have `stub_column`, why is it being fetched again (as `self._d[stub_index]`)?